packages/analytics/amplify_analytics_pinpoint/android/src/main/kotlin/com/amazonaws/amplify/amplify_analytics_pinpoint/amplify_analytics_pinpoint/AmplifyAnalyticsPinpointPlugin.kt

```diff
@@ -0,0 +1,40 @@
+
+package com.amazonaws.amplify.amplify_analytics_pinpoint.amplify_analytics_pinpoint
```

Comment:
License header

Reply:
I think I made some changes but didn't commit them hence the issues that were resolved but not actually fixed here ... 

Likely due to some rebase work I did I might have missed a commit here